pom.xml

```diff
@@ -1443,25 +1443,6 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.commons</groupId>
-                <artifactId>commons-vfs2</artifactId>
-                <version>2.0</version>
```

Comment:
this was meant to exclude certain transitive deps from accumulo, but we didn't update the version with accumulo 1.7 which now imports 2.1. this caused some mock tests to fail.